.github/workflows/docker-image.yml

```diff
@@ -3,83 +3,79 @@ name: Push to Docker Hub
 on: [workflow_dispatch]
 
 jobs:
-
   backend-image:
     name: Build backend image
     runs-on: ubuntu-latest
 
     steps:
-      - 
-        name: ☁️ Checkout source
+      - name: ☁️ Checkout source
         uses: actions/checkout@v3
-      -
-        name: 🔧 Set up QEMU
+      - name: 🔧 Set up QEMU
         uses: docker/setup-qemu-action@v2
-      -
-        name: 🔧 Set up Docker Buildx
+      - name: 🔧 Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
-      -
-        name: 🐋 Login to Docker Hub
+      - name: 🐋 Login to Docker Hub
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      # -
-      #   name: 📦 Build backend and export to Docker
-      #   uses: docker/build-push-action@v3
-      #   with:
-      #     load: true
-      #     context: backend
-      #     tags: infisical/backend:test
-      # -
-      #   name: 🧪 Test backend image
-      #   run: |
-      #     docker run --rm infisical/backend:test
-      -
-        name: 🏗️ Build backend and push
+      - name: 📦 Build backend and export to Docker
+        uses: docker/build-push-action@v3
+        with:
+          load: true
+          context: backend
+          tags: infisical/backend:test
```

Comment:
Double checking, but this creates the image with tag `test` locally right?

Reply:
actaully, i see you have `load: true` so it only exports it locally

Reply:
Yup, build and tag `test`. But load only so we can use it. On succeeding build (with push), image will be reused but tagged as `latest`.

Reply:
Awesome